src/pudl/metadata/classes.py

```diff
@@ -572,6 +573,24 @@ class Field(PudlMeta):
     harvest: FieldHarvest = FieldHarvest()
     encoder: Encoder | None = None
 
+    @classmethod
```

Comment:
PUDL models schemas are stored as `pyarrow` schemas, which are converted to our standard metadata structures here.